projects/ad7616_sdz/zed/system_project.tcl

```diff
@@ -31,21 +31,17 @@ source $ad_hdl_dir/projects/scripts/adi_board.tcl
 #
 ##--------------------------------------------------------------
 
-if {[info exists ::env(SER_PAR_N)]} {
-  set S_SER_PAR_N [get_env_param SER_PAR_N 0]
-} elseif {![info exists SER_PAR_N]} {
-  set S_SER_PAR_N 1
-}
+set INTF [get_env_param INTF 0]
```

Comment:
In the text it says that the default value is 1 (serial). How is it actually?

Reply:
Parallel mode.